### PR TITLE
Fix missing string with stringtable

### PR DESCRIPTION
**Pull request by GodofMonkeys**
Fix Advanced Fatigue & QuickMount missing, add string into stringtable, and update Chinese localization with this two pbo.

### CODE REVIEW

File: addons/quickmount/ACE_Settings.hpp [hunk 1]
```diff
@@ -36,6 +36,6 @@ class ACE_Settings {
         description = CSTRING(PriorityDescription);
         isClientSettable = 1;
         force = 0;
-        values[] = {"Driver", "Gunner", "Commander", "Passenger"};
```
Comment (jonpas) [inline addons/quickmount/ACE_Settings.hpp]: There must be vanilla translations for those already.

Comment (GodofMonkeys) [inline addons/quickmount/ACE_Settings.hpp]: It is vanilla translations? But I can't see it translation in my game.

Image
alt text

Comment (GodofMonkeys) [inline addons/quickmount/ACE_Settings.hpp]: Oh, I see. You mean I should use vanilla game's string to replace these string?

**Comment by GodofMonkeys** [comment]
Add more, fix Gestures & DAGR missing string

File: addons/quickmount/ACE_Settings.hpp (hunk 1 above)
Comment (jonpas) [inline addons/quickmount/ACE_Settings.hpp]: Yeah, but I only found 2:
- Driver: `str_a3_cfgvehicles_c_driver_1_random_base_f0`
- Commander: `str_a3_cfgvehicles_highcommand_moduledescription_anybrain_0`

Might be better to just translate them but put them in `common` component so they can be used in more places.

Comment (GodofMonkeys) [inline addons/quickmount/ACE_Settings.hpp]: I find it. Waite me update.

File: addons/quickmount/ACE_Settings.hpp [hunk 2]
```diff
@@ -36,6 +36,6 @@ class ACE_Settings {
         description = CSTRING(PriorityDescription);
         isClientSettable = 1;
         force = 0;
-        values[] = {CSTRING(PriorityDriver), CSTRING(PriorityGunner), CSTRING(PriorityCommander), CSTRING(PriorityPassenger)};
+        values[] = {"$str_getin_pos_driver", "$str_getin_pos_gunn", "$str_getin_pos_comm", "$STR_GETIN_POS_PASSENGER"};
```
Comment (GodofMonkeys) [inline addons/quickmount/ACE_Settings.hpp]: This is might be better :)

Comment (jonpas) [inline addons/quickmount/ACE_Settings.hpp]: Awesome!

**Comment by GodofMonkeys** [comment]
I merge with @Dissyu.

**Pull request closed without merging**